### DISCUSSION TITLE
关于grp

### DISCUSSION THREAD

**Discussion by Pragmatism0220** [Equim-chan/Mortal#39] [Q&A]
其实一直不大理解这个参数……只是机械化地训练了一下
可以解释一下grp模型是起到什么作用嘛……实在不是很理解，谢谢！

**Comment by Equim-chan** [comment]
Mortal 中的 GRP 在功能上和 Suphx 论文中的提到的 global reward predictor 是一样的。

**Reply by Pragmatism0220** [reply]
那么这个grp模型的训练时间对grp模型的性能影响会多少影响到state模型的性能呢？一般需要大概训练多少step呢？注意到grp的实现里面是一个shuffle的cycle。

**Reply by Pragmatism0220** [reply]
可能我没说清楚，我想表达的意思就是，在模型收敛后，grp是作为全局奖励的。那是否应该精确地训练出一个优秀的grp模型，从而提升state模型的性能？那这样进行更长时间的迭代，在可能的过拟合之前，会对state模型有什么样的影响吗？还是说，差不多就行了，对state模型的影响因素更多在训练state模型本身？

**Reply by Equim-chan** [reply]
我没有在训练 Mortal 时控制过 GRP 模型这个变量。我只是以训练常规机器学习模型的方式训练的 GRP 模型，即尽量降低 val loss 就完事了。

**Reply by Nitasurin** [reply]
@Equim-chan 您的GRP模型训练完成后准确率和loss大概是多少？GRP在较大的数据集上似乎很难获得一个比较好的准确率和loss

**Reply by Equim-chan** [reply]
@Niitsune 我这边 val acc 0.23 左右

**Reply by Iriya99** [reply]
请问grp和dqn大概需要多少训练数据，以及训练多少step呢
